rest/route/annotations.go

```diff
@@ -271,6 +272,21 @@ func (h *annotationByTaskPutHandler) Parse(ctx context.Context, r *http.Request)
 			StatusCode: http.StatusBadRequest,
 		}
 	}
+
+	catcher := grip.NewBasicCatcher()
+	for _, issue := range h.annotation.Issues {
+		catcher.Add(annotations.ValidateIssueURL(*issue.URL))
```

Comment:
Would you consider using [FromStringPtr](https://github.com/evergreen-ci/evergreen/blob/822f3633bfa1e9f8c95d49f91c602e0b65bb7072/rest/model/string.go#L7)? It handles the case where URL is nil.